pkg/cmd/taskrun/delete.go

```diff
@@ -17,14 +17,15 @@ package taskrun
 import (
 	"errors"
 	"fmt"
+	taskrun "github.com/tektoncd/cli/pkg/taskrun/sort"
```

Comment:
Can you rename this to `trsort` to keep it consistent with how it's referenced else where?

Reply:
Done